pkg/setting/setting.go

```diff
@@ -838,6 +797,14 @@ func (cfg *Cfg) Load(args *CommandLineArgs) error {
 	return nil
 }
 
+func readSectionValueString(section *ini.Section, keyName string, defaultValue string) string {
+	if !section.HasKey(keyName) {
+		return defaultValue
+	}
```

Comment:
Doesn't `section.Key(keyName).MustString(defaultValue)` have the same effect as this?